src/test/java/com/coravy/hudson/plugins/github/GithubProjectPropertyTest.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2015 CloudBees, Inc.
```

Comment:
btw, @oleg-nenashev introduced chaos into this plugin, root license is MIT, but pom defines apache. Could you remove license header for now? Or it required by your company rules?